storage/mongodb/files/scripts/test-mongo.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -e
+
+readonly NAMESPACE=${1}
+readonly ITERATION=${2}
+readonly THREADS=${3}
+readonly WORKLOAD=${4}
+
+output_dir=$(dirname $0)
+
+readonly MONGODB_IP=$(oc get svc -n ${NAMESPACE} | grep mongodb | awk '{print $3}')
```

Comment:
I would change this to ```grep "^mongodb" ````

Reply:
Change to 
readonly MONGODB_IP=$(oc get svc -n ${NAMESPACE} | grep -v glusterfs | grep mongodb | awk '{print $3}')

Hope that is OK too.

Reply:
yes, that is fine